src/components/buttons/secondary-button/secondary-button.js

```diff
@@ -132,6 +132,14 @@ SecondaryButton.propTypes = {
         `
       );
     }
+    if (props.to && props.type !== 'button') {
+      throw new Error(
+        oneLine`
+          ${componentName}: "${propName}" does not have any effect when
+          "to" is set.
+        `
+      );
+    }
```

Comment:
Oh wow. Malcolm did it :)

Reply:
...and now he goes "Why did I oneline this after all?"

Reply:
🙃